docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,777 @@
+
+This is the output you should see after installing the templates in a machine that has the .NET Aspire Workload installed:
+
+```dotnetcli
+> dotnet new install Aspire.ProjectTemplates::9.0.0-rc.1.24511.1 --force
+The following template packages will be installed:
+   Aspire.ProjectTemplates::9.0.0-rc.1.24511.1
+
+Installing the template package(s) will override the available template package(s).
+The following template package(s) are already available:
+   Aspire.ProjectTemplates::8.2.1
+
+Success: Aspire.ProjectTemplates::9.0.0-rc.1.24511.1 installed the following templates:
+Template Name                    Short Name                                       Language  Tags
+-------------------------------  -----------------------------------------------  --------  -------------------------------------------------------
+.NET Aspire 8 App Host           aspire-apphost-8                                 [C#]      Common/.NET Aspire/Cloud
+.NET Aspire 8 Empty App          aspire-8                                         [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
+.NET Aspire 8 Service Defaults   aspire-servicedefaults-8                         [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
+.NET Aspire 8 Starter App        aspire-starter-8                                 [C#]      Common/.NET Aspire/Blazor/Web/Web API/API/Service/Cloud
+.NET Aspire 8 Test Project (...  aspire-mstest-8                                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service/Test
+.NET Aspire 8 Test Project (...  aspire-nunit-8                                   [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service/Test
+.NET Aspire 8 Test Project (...  aspire-xunit-8                                   [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service/Test
+.NET Aspire 9 App Host           aspire-apphost,aspire-apphost-9                  [C#]      Common/.NET Aspire/Cloud
+.NET Aspire 9 Empty App          aspire,aspire-9                                  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
+.NET Aspire 9 Service Defaults   aspire-servicedefaults,aspire-servicedefaults-9  [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service
+.NET Aspire 9 Starter App        aspire-starter,aspire-starter-9                  [C#]      Common/.NET Aspire/Blazor/Web/Web API/API/Service/Cloud
+.NET Aspire 9 Test Project (...  aspire-mstest,aspire-mstest-9                    [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service/Test
+.NET Aspire 9 Test Project (...  aspire-nunit,aspire-nunit-9                      [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service/Test
+.NET Aspire 9 Test Project (...  aspire-xunit,aspire-xunit-9                      [C#]      Common/.NET Aspire/Cloud/Web/Web API/API/Service/Test
+```
+
+Now you have the .NET Aspire 9 templates installed on your machine. You can create a new .NET Aspire 9 project by running the following command:
```

Comment:
We use the versions `9.0` and `9` for Aspire inconsistently throughout the document. Which should it be? Most other examples seem to be 9.0.

```suggestion
Now you have the .NET Aspire 9.0 templates installed on your machine. You can create a new .NET Aspire 9.0 project by running the following command:
```

Reply:
The first time we talk about it, we should say ".NET Aspire 9.0". Then after that, we should say ".NET Aspire 9" for simplicity.

Reply:
Makes sense. There are a bunch of places to fix up then :)